integration-tests/src/test/resources/features/AccessTokenAPI.feature

```diff
@@ -1,19 +1,41 @@
 Feature: Access Token API
 
-  Background: a session id is returned
+  Scenario Outline: access token is returned
     Given authorization JAR for test user 681
+    And AccessToken lambda implementation is in '<SessionLambdaImplementation>'
```

Comment:
`Given authorization JAR for test user 681`
```suggestion
    And Session lambda implementation is in '<SessionLambdaImplementation>'
```

All these lines that are paired with `authorization JAR` should be referring to the Session lambda